generator/main.go

```diff
@@ -0,0 +1,62 @@
+/*
+Copyright 2020 The Kubernetes Authors.
```

Comment:
Is this intentional?

Reply:
yeah, because I have copied almost the whole file from Kubernetes repo :) So I decided to leave it as is